recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set version = "1.27.28" %}
-{% set hash = "b00da6290d8d42c6c3048b045d6537421e8b83e03b4f24bb44c5ea9f37ab6258" %}
+{% set version = "1.27.59" %}
+{% set hash = "eda4aed6ee719a745d1288eaf1beb12f6f6448ad1fa12f159405db14ba9c92cf" %}
 
 package:
   name: botocore
```

Comment:
Can be updated:
```
{% set name = "botocore" %}
{% set version = "1.27.59" %}
{% set hash = "eda4aed6ee719a745d1288eaf1beb12f6f6448ad1fa12f159405db14ba9c92cf" %}

package:
  name: {{ name|lower }}
  version: {{ version }}
```
see a `grayskull` web-variant: https://www.marcelotrevisani.com/grayskull

Reply:
This has been updated